app/models/goal.rb

```diff
@@ -1,3 +1,14 @@
 class Goal < ActiveRecord::Base
+
+  TIME_ALLOTTED_TO_COMPLETE_A_GOAL_IN_DAYS  = 7
+
   belongs_to :user
+
+  def self.active
+    where("created_at > ?", TIME_ALLOTTED_TO_COMPLETE_A_GOAL_IN_DAYS.days.ago)
+  end
+
+  def self.expired
+    where("created_at <= ?", TIME_ALLOTTED_TO_COMPLETE_A_GOAL_IN_DAYS.days.ago)
```

Comment:
`TIME_ALLOTTED_TO_COMPLETE_A_GOAL_IN_DAYS.days.ago` is used in a couple places, consider extracting to a private method within the `Goal` class e.g.

```ruby
private

def self.created_at_expiration_boundary
   TIME_ALLOTTED_TO_COMPLETE_A_GOAL_IN_DAYS.days.ago
end
```

and using the method within the `active` and `expired` methods.